src/then/then.js

```diff
@@ -22,10 +22,28 @@ const execPromise = (retValue, error) => {
   return new Promise(resolve => resolve(retValue));
 };
 
+/*
+Hi everyone, I'm the Then class! I have two jobs:
+1) Store values to return later
+2) Return the values when asked
+*/
 class Then {
   constructor(mockedFunction){
     this.force = false;
+    this.isPromise = false;
     this.mockedFunction = mockedFunction;
+    this.promise = {
```

Comment:
Promise could be moved to its own object. Main reason I didn't is to leverage lexical `this`. But the current `then` could also be passed into the constructor of the `promise` object, as we are doing with `mockedFunction`